src/Bowling.cpp

```diff
@@ -64,3 +65,37 @@ void Bowling::readDirectory(path p)
         std::cout << ex.what() << '\n';
     }
 }
+
+std::string Bowling::toString() const
+{
+    std::stringstream ss;
+    std::string data;
+    for (int i = 0; i < getNumberOfLanes(); i++)
+    {
+        Lane lane = getLane(0);
+        std::string laneData;
+        std::stringstream no; 
+        no << ++i;
+        std::string numberOfLane = no.str();
+            laneData.insert(laneData.size(), "### Lane ");
+            laneData.insert(laneData.size(), numberOfLane);
+            laneData.insert(laneData.size(), ": ");
+            laneData.insert(laneData.size(), "GAME STATUS"); // game status method is needed
+            laneData.insert(laneData.size(), " ###\n");
```

Comment:
Maybe better solution will be `ss << "### Lane " << (i+1) << ": " << "GAME STATUS" << " ###\n"`